source/install/desktop-app-changelog.rst

```diff
@@ -3,12 +3,89 @@ Desktop Application Changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v4.7 <#id1>`_
-- `Release v4.6 <#id4>`_
-- `Release v4.5 <#id16>`_
-- `Release v4.4 <#id32>`_
-- `Release v4.3 <#id51>`_
-- `Release v4.2.3 <#id68>`_
+- `Release v5.0 <#id1>`_
+- `Release v4.7 <#id4>`_
+- `Release v4.6 <#id25>`_
+- `Release v4.5 <#id37>`_
+- `Release v4.4 <#id53>`_
+- `Release v4.3 <#id72>`_
+
+Release v5.0
+--------------
+
+**Release Date: 2021-10-13**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop Apps are required to be used with any `supported Extended Support Release or a newer Mattermost server version <https://docs.mattermost.com/upgrade/release-lifecycle.html>`_.
+
+Breaking Changes / Upgrade Notes
+~~~~~~~~~~~~~~~
+
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Added new branding for Desktop App v5.0.
+
```

Comment:
Also add

Automatic Updates

Desktop v5.0 will prompt users to download and directly install new version updates when Mattermost restarts. This can be disabled using Group Policy settings (link to docs TBD)

Reply:
Will add once merged.